flyteadmin/tests/bootstrap.go

```diff
@@ -19,8 +20,18 @@ const insertExecutionQueryStr = `INSERT INTO "executions" ` +
 	`("execution_project","execution_domain","execution_name","phase","launch_plan_id","workflow_id") ` +
 	`VALUES ('%s', '%s', '%s', '%s', '%d', '%d')`
 
+const sandboxEnv = "USE_LOCAL_SANDBOX"
+
 var adminScope = promutils.NewScope("flyteadmin")
 
+func getDbConfigWithEnv() *database.DbConfig {
+	if os.Getenv(sandboxEnv) == "False" {
```

Comment:
What if we use sandbox config by default and only use the k8s integration test config if explicitly instructed to from the env var?

Reply:
Yes this is the current behavior. It will use sandbox config unless you set USE_LOCAL_SANDBOX as false

Reply:
Actually thinking about it some more, I think it might be more intuitive to assume a default bool value where possible (false). So rather than checking for Falsiness can we by default check to see if "USE_INTEGRATION_TEST_CONFIG" is set and _true_ and then use that specific config?